src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -253,9 +252,21 @@ public String sendArtefactForSubscription(Artefact artefact) {
     /**
      * Scheduled method that checks daily for newly dated from artefacts.
      */
-    @Scheduled(cron = "${cron.daily-display-from}")
+    @Scheduled(cron = "${cron.daily-start-of-day}")
     public void checkNewlyActiveArtefacts() {
         List<Artefact> newArtefactsToday = artefactRepository.findArtefactsByDisplayFrom(LocalDate.now());
         newArtefactsToday.forEach(artefact -> log.info(sendArtefactForSubscription(artefact)));
     }
+
+    /**
+     * Scheduled method that checks daily for newly outdated artefacts based on a yesterday or older display to date
+     */
+    @Scheduled(cron = "${cron.daily-start-of-day}")
+    public void deleteExpiredBlobs() {
+        List<Artefact> outdatedArtefacts = artefactRepository.findOutdatedArtefacts();
+        outdatedArtefacts.forEach(artefact -> log.info(azureBlobService.deleteBlob(artefact.getSourceArtefactId(),
+                                                                   artefact.getProvenance())));
+        artefactRepository.deleteAll(outdatedArtefacts);
```

Comment:
We should also delete the data from the blob store too

Reply:
thats what line 267 is doing

Reply:
Ah yep, missed that!